ironfish/src/primitives/noteEncrypted.ts

```diff
@@ -39,16 +42,14 @@ export class NoteEncrypted {
     // ephemeral public key
     reader.seek(32)
     // encrypted note
-    reader.seek(72)
+    reader.seek(ENCRYPTED_NOTE_PLAINTEXT_LENGTH)
     // aead MAC
-    reader.seek(16)
+    reader.seek(MAC_LENGTH)
     // note encryption keys
-    reader.seek(64)
-    // aead MAC
-    reader.seek(16)
+    reader.seek(NOTE_ENCRYPTION_KEY_LENGTH)
```

Comment:
I think this should stay split out into ENCRYPTED_SHARED_KEY_SIZE (64) + MAC_LENGTH (16), since we also split out the ENCRYPTED_NOTE_PLAINTEXT_LENGTH from the MAC_LENGTH.